bundles/org.openhab.binding.ecotouch/src/main/java/org/openhab/binding/ecotouch/internal/EcoTouchConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ecotouch.internal;
+
+/**
+ * The {@link EcoTouchConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Sebastian Held - Initial contribution
+ */
+public class EcoTouchConfiguration {
+
+    /**
+     * Sample configuration parameter. Replace with your own.
+     */
+    public String ip;
+    public String username;
+    public String password;
+    public Integer refresh;
```

Comment:
```suggestion
@NonNullByDefault
public class EcoTouchConfiguration {
    public String ip="";
    public String username="";
    public String password="";
    public Integer refresh=0;
```

All classes should be @NonNullByDefault
Good practice to initilize all variables at creation so they cant be null.
Remove all comments that are auto generated.